inst/templates/contributing-template.txt

```diff
@@ -106,13 +97,10 @@ you may want to look at
 [How to Contribute to an Open Source Project on GitHub][how-contribute].
 In brief:
 
-1.  The published copy of the lesson is in the `gh-pages` branch of the repository
-    (so that GitHub will regenerate it automatically).
+1.  The published copy of the lesson is in the `main` branch of the repository,
+    that branch should be protected from direct commits.
     Please create all branches from that,
-    and merge the [master repository][repo]'s `gh-pages` branch into your `gh-pages` branch
-    before starting work.
-    Please do *not* work directly in your `gh-pages` branch,
-    since that will make it difficult for you to work on other contributions.
+    discuss with the maintainer about what branch to use before starting to work.
```

Comment:
I believe this entire sentence can be deleted since GitHub flow is described below.